docs/search-omero.rst

```diff
@@ -66,6 +66,55 @@ Step-by-Step
 
 #. Click on the Browse link |image2|\ in one search result line of the last image (in the right-hand part of the centre pane) to navigate back to the main webclient.
 
+.. note::
+    Spaces, stars, question marks and ``^`` characters are to be avoided in the Keys.
+    If you already have these characters in Keys in your OMERO, 
+    then try to replace them with underscores if you want to 
+    use the Search functionality on them.
+    Spaces, stars, question marks and ``^`` characters in Values 
+    are acceptable, but should be avoided if possible, see examples below.
+
+Key-Value pairs search examples
+-------------------------------
+
+Considering the following setup of 12 separate images (image 1 through 12):
+
+1. Key ``GFP H2B`` with Value ``2 uM`` on image 1
+2. Key ``GFP^H2B`` with Value ``2 uM`` on image 2
+3. Key ``H2B`` with Value ``2`` on image 3
+4. Key ``H2B`` with Value ``4`` on image 4
+5. Key ``GFP-H2B`` with Value ``2-uM`` on image 5
+6. Key ``GFP-H2B`` with Value ``2 uM`` on image 6
+7. Key ``GFP_H2B`` with Value ``2_uM`` on image 7
+8. Key ``GFP^H2B`` with Value ``2^uM`` on image 8
+9. Image 9 named ``GFP`` with no Key-Value pairs
+10. Image 10 named ``uM`` with no Key-Value pairs
+11. Image 11 named ``H2B`` with no Key-Value pairs
+12. Image 12 named ``2`` with no Key-Value pairs
+
+Basic **Search** tab:
+   - ``GFP H2B:2 uM`` finds images 1,2,3,5,6,7,8,9,10. In that case, the query is interpreted as ``GFP`` OR ``H2B:2`` OR ``uM``.
+   - ``"GFP H2B":2 uM`` throws an error. Do not use quotes around Key names!
```

Comment:
That this throws an error seems like quite the bug/limitation.

Reply:
@joshmoore created https://github.com/ome/omero-web/issues/373